PR for issue Dap-20

Danielle0727 opened the pull request:
I added two images in Section 2 and named them "ExcelAND.jpg" and "ExcelAVERAGEIF.jpg". These were images that were included in the PPT. I also created a table for the Functions. Please let me know if you do not want them in table format or have suggestions on how to make is easier to view. I welcome any and all feedback, especially since this is my first time doing markdown for the curriculum.

JohanBester commented on Section02/2.3-DAPandExcel.md:
```diff
@@ -0,0 +1,205 @@
+## <div align="center">Remember Data Wrangling?</div>
+
+<div align="center">The process of cleaning and unifying messy and complex data sets for easy access and analysis.</div>
+# <div align="center">&#8681;</div>
```
I suggest we not center these sections

JohanBester commented on Section02/2.3-DAPandExcel.md:
```diff
@@ -0,0 +1,205 @@
+
+We will use the following spreadsheet together to learn more about Excel.
+
+![Excel_Practice_Student.xlsx](./activities/Excel_Practice_Student.xlsx)
```
This link not working as this is setup as an image. Removing the exclamation mark will make the link work

Danielle0727 replied:
Removed exclamation mark

JohanBester commented on Section02/2.3-DAPandExcel.md:
```diff
@@ -0,0 +1,205 @@
+# <div align="center">&#8681;</div>
+<div align="center">Organizing and processing data</div>
+
+![Data Wrangling](./img/DataWranglingSteps.png)
```
Often layout is enhanced by adding a break mark on a blank line between sections like here below 

<br>

Danielle0727 replied:
I completely agree! I added further breaks to the entire document.

JohanBester commented on Section02/2.3-DAPandExcel.md:
```diff
@@ -0,0 +1,205 @@
+| **AND** | **AND USING IF** |
+|:-------:|:----------------:|
+|`=AND=AND(**[logical1], [logical2], ...**)`|`=IF(AND(**logical_test, logical_test), [“value_if_true”], [“value_if_false”]**)`|
+|<font color=blue>If a number is greater than or smaller than another number or is equal to something.</font> | Produces whatever outcome you need it to read if you don’t want “True” or “False” … |
```
I suggest we shy away from using dark blue due to poor contrast issue in dark mode environments. Kindly please change the color of just use the default text color.

![Screenshot 2023-01-10 170249](https://user-images.githubusercontent.com/29065848/211680574-1744a55c-2a51-4b73-906a-5b120a823de8.png)

Danielle0727 replied:
I removed the color in that area, as well as under VLOOKUP. Since I removed the color under VLOOKUP, I added in paranthesis what each description is referring to, in order to make it easier for the students. In the excel sheet, I believe there were colors to help identify what each description was referring to, but please let me know if you would like me to remove my addition.

JohanBester commented on Section02/2.3-DAPandExcel.md:
```diff
@@ -0,0 +1,205 @@
+|**<font color=blue>table_array</font>**   (required) lookup range,|
+|**col_index_num**    (required) of lookup_value,|
+|**<font color=purple>Approximate - 1/TRUE or Exact - 0/FALSE</font>**|
+
```
I hate suggesting this as it will create additional work. However, after looking at this online and offline, it might be beater if the functions are not put in a table.
Instead just one after another from top down type of layout would be easier on the eye.

Danielle0727 replied:
No worries! It's all a learning process and I don't mind fixing it. I want to make it easy to read for the students.

JohanBester commented on Section02/2.3-DAPandExcel.md:
```diff
@@ -0,0 +1,205 @@
+A summary of a large dataset that usually includes the total figures, average, minimum, maximum, etc.
+
+<font color = darkblue>Open Up Sales Spreadsheet</font>
+
```
Again I would advise against using dark font because of possible conflicts with dark mode environments

![Screenshot 2023-01-10 170741](https://user-images.githubusercontent.com/29065848/211681339-42168d8f-c942-4759-b167-b87cd4bb3284.png)

Danielle0727 replied:
Thank you Johan for all of the suggestions and feedback. I made all of the changes you suggested, and realized I may have overdone it on my replies and I could have just sent one reply.

JohanBester reviewed (changes requested):
Hi Danielle,
You did great!
I did leave several comments regarding style more than anything else.
Also a comment about the Excel Student worksheet link that does not show at this stage.
Let me know if there are any questions or you need clarification on any point.

JohanBester commented on Section02/2.3-DAPandExcel.md:
```diff
@@ -0,0 +1,305 @@
+<br>
+
+**AND**  
+- `=AND=AND([logical1], [logical2], ...)`
```
I think the slide had an error here.
I believe that line should just be `=AND([logical1], [logical2], ...)`
Sorry I missed this the 1st time

JohanBester commented on Section02/2.3-DAPandExcel.md:
```diff
@@ -0,0 +1,305 @@
+**AVERAGEIFS**
+- Calculates the average of a range based on one or more true/false conditions
+- `=AVERAGEIFS(average_range, criteria_range1, criteria1, ...)`
+
```
I just noticed that this section is a little confusing.
I want to propose a little reorganization if you don't mind please.
I suggest this order:-

Delete this line
(  ### =AVERAGEIFS(C27:C38, B27:B38, F28, D27:D38,G28)  )

**AVERAGEIFS**
- Calculates the average of a range based on one or more true/false conditions
- `=AVERAGEIFS(average_range, criteria_range1, criteria1, ...)`
- `=AVERAGEIFS(C27:C38, B27:B38, F28, D27:D38,G28)`

`<br>`

**AVERAGEIF**  
- Combine averages from different cells
- `=AVERAGEIF(range, criteria, [average_range])`

![Screenshot 2023-01-11 191634](https://user-images.githubusercontent.com/29065848/211952784-0c101073-aadf-4997-a76b-387c5654ef94.png)

JohanBester reviewed (changes requested):
The changes you made are great.
I made a few more suggestions if you don't mind.
Also, maybe add a small Intro at the top of the section, and a small summary at the end.
Thanks!  :)

Danielle0727 commented:
-Removed the extra AND in =AND([logical1], [logical2], ...)
-Made updates to the AVERAGEIF(S) feedback. 
-Updated the AVERAGEIF image. 
-Added an Objective and Summary section.
